CHANGELOG.md

```diff
@@ -1,15 +1,28 @@
 # vscode-powershell Release History
 
-## v1.12.0
-### Friday, March 29, 2019
+## v1.12.1
+### Wednesday, April 10, 2019
+
+#### [vscode-powershell](https://github.com/powershell/vscode-powershell)
 
-### Highlights
+- [vscode-PowerShell #1852](https://github.com/PowerShell/vscode-PowerShell/pull/1852) -
+  Turn `powershell.codeformatting.useCorrectCasing` setting off by default until PSSA issues are fixed (Thanks @bergmeister!)
+- [vscode-PowerShell #1837](https://github.com/PowerShell/vscode-PowerShell/pull/1837) -
+  Do not use -EncodedCommand on Windows
```

Comment:
Shouldn't the changelog mention the switch to camelCase on those two codeFormatting settings?

Reply:
Yeah I'll change it when that PR goes in